config/locales/en.yml

```diff
@@ -147,6 +147,50 @@ en:
         withdrawn: Trainee withdrawn
         deferred: Trainee deferred
         qts_awarded: Trainee awarded QTS
+        programme_details: Programme details not started
+    sections:
+      personal_details:
+        not_started: 
+          title: Personal details not started
+          link_text: Start section
```

Comment:
Could we have just one translation for `link_text.not_stated` and `link_text.in_progress`, seeing as they're always either 'Start section' or 'Continue section'?

Seems like a lot of repetition if we're not expecting them to individually change.